7_config_merge/out/nml_list.rds.ind

```diff
@@ -1,2 +1,2 @@
-hash: fb6583cfd0c6e2f07880b8d2be49ed4a
+hash: bac2fe415108be5de85a36345a80ec11
```

Comment:
But when I compare what I get from `sc_retrieve()` with what I had locally, they are the same. So must be fine?

```r
before <- readRDS('7_config_merge/out/nml_list.rds')
after <- readRDS(scipiper::sc_retrieve('7_config_merge/out/nml_list.rds.ind'))
all.equal(before, after)

[1] TRUE
```

Reply:
Yes, can confirm this is fine. I had to rebuild this target locally because of the [issue you mentioned above](https://github.com/USGS-R/lake-temperature-model-prep/pull/333#pullrequestreview-947797204).

RE: this comment and the linked comment above - I had a sidebar with @jread-usgs about this issue earlier this morning. Here is the general summary: I had to rebuild `7_config_merge/out/nml_list.rds.ind` because I did not have it locally and it is required to complete [`count_lake_obs`](https://github.com/USGS-R/lake-temperature-model-prep/blob/3430117a2f98b99a3022d80b0bc6a27ded43ccb5/8_viz/src/count_lake_obs.R#L8) which gets called when you run `scmake('8_viz')`. `count_lake_obs` is brittle specifically because of [this](https://github.com/USGS-R/lake-temperature-model-prep/blob/3430117a2f98b99a3022d80b0bc6a27ded43ccb5/8_viz/src/count_lake_obs.R#L8) line where the `as_data_file` function is used instead of `sc_retreive`. Jordan and I discussed that it would be easy enough to fix that in a separate, small PR. But, I did not open an issue 😬 !

As for the part on `nml_list.rds`: I chose to rebuild `7_config_merge/out/nml_list.rds.ind` because, when I ran `sc_retrieve('7_config_merge/out/nml_list.rds.ind')` it only returned the file name `7_config_merge/out/nml_list.rds` in the console (instead of retrieving the object). I am not sure what caused this unexpected issue, nor did I dig into it too deeply. I chose to make it rebuild instead (maybe this was not the best decision, in retrospect).

Finally, the discrepancies between our two versions of `8_viz/out/lake_obs_counts.png`: oh no! I think we will need to discuss this tomorrow. I need advice on how best to tackle this one.

Reply:
@lindsayplatt also, for some reason I cannot reply to your first comment so I tackled everything here